### PR TITLE
Reworked webui reader

### PR DESCRIPTION
**Pull request by matthew73210**
Reworked the webui.log reader. Placed the functions it what seems the right folders. #153 & #138 
Couldn't for the life of me get the templating working like settings, just created a work around in logs.html. Pretty much just index.html copied and removed some parts.
Need to work on making it dynamic as it depends on refreshing the browser.
Being a gitnoob i commited my changes to config, can ignore them.

Might need to be carefull about CSS attacks cus the function reads the webui log and posts as is, if someones puts some code in there ? I think code rabbit will flag it up again.

### CODE REVIEW

**Comment by matthew73210** [comment]
The webui doesn't have much going on, but the basis is there to read the other logfile too. Might add two links under logs. Or just paste all logs to webui.log and have a single one?

File: .gitignore
Comment (tphakala) [inline .gitignore:1]: Please do not include this in in pull request, this is unnecessary change

File: internal/conf/config.yaml
Comment (matthew73210) [inline internal/conf/config.yaml:1]: Not sure what's going on here.

File: .gitignore
Comment (matthew73210) [reply .gitignore:1]: Sorry my bad.

**Comment by tphakala** [comment]
abdandoning this for now
